cmd/helm3.go

```diff
@@ -129,6 +129,10 @@ func (d *diffCmd) template(isUpgrade bool) ([]byte, error) {
 		flags = append(flags, "--is-upgrade")
 	}
 
+	if disableOpenAPIValidation {
```

Comment:
You should just use `if d.disableOpenAPIValidation {` instead of passing the extra arg and referencing it

Reply:
@mumoshu Thank you for your review. Fixed and resolved conflicts.